src/annotator/test/pdf-sidebar-test.js

```diff
@@ -0,0 +1,238 @@
+import $ from 'jquery';
+
+//import events from '../../shared/bridge-events';
+
+import PdfSidebar from '../pdf-sidebar';
+import { $imports } from '../pdf-sidebar';
```

Comment:
nit these 2 lines could be 
`import PdfSidebar, { $imports } from '../pdf-sidebar';`

Reply:
edit: I think we can just remove $imports entirely.